app/views/candidate_mailer/nudge_unsubmitted_with_incomplete_references/no_references.text.erb

```diff
@@ -1,24 +1,20 @@
 Dear <%= @application_form.first_name %>
 
-You have not requested your teacher training references yet. 
+You have not added your teacher training references yet.
 
-Before you can submit your application, you need to receive 2 references.
+You'll need to give details of 2 people who can give you a reference. They'll only be contacted if you accept an offer on a course.
 
 [Sign in to your account to start requesting references](<%= candidate_magic_link(@application_form.candidate) %>).
 
 # Who to choose as a referee
 
 A referee could be someone like a manager, a university tutor or a client.
 
-In their reference, your referee will be asked to talk about your:
+In their reference, your referee will be asked to:
 
-  - communication skills
-  - reliability and professionalism
-  - transferable skills
-  - ability to work with children
-  - academic skills
-
-So choose referees who could comment on as many of these as possible.
+  - confirm how they know you and how long they’ve known you for
+  - state whether they have any concerns about you working with children
+  - give information which can be used to check your application
```

Comment:
Could drop all this, or just simplify it to "They’ll be asked if they know any reason why you should not work with children." to match the web interface?

Reply:
I'd be happy mirroring the web interface - will defer to @johndoates. I lifted this from the referee request emails as that was 'approved' content.

Reply:
I've made a couple of suggestions to make it better match the latest wording in other places. 

I wondered whether we need the 'who to choose' section at all. There could be some benefit if the candidate is hesitant to click through because they don't have someone in mind, but we aren't giving them all the details. For example, they may have a couple of professionals in mind but when they click through they discover they need an academic referee.